crates/contracts/src/options/source_simf/options.simf

```diff
@@ -116,9 +116,9 @@ fn funding_path(token_branch: Either<(), ()>, expected_asset_amount: u64) {
     let grantor_token_amount: u64 = unwrap_right::<(u1, u256)>(unwrap(unwrap(jet::issuance_asset_amount(1))));
     assert!(jet::eq_64(option_token_amount, grantor_token_amount));
 
-    divmod_eq(collateral_amount, param::CONTRACT_SIZE, option_token_amount);
-    divmod_eq(collateral_amount, param::ASSET_STRIKE_PRICE, expected_asset_amount);
-    divmod_eq(expected_asset_amount, param::GRANTOR_TOKEN_STRIKE_PRICE, grantor_token_amount);
+    divmod_eq(collateral_amount, param::COLLATERAL_PER_CONTRACT, option_token_amount);
+    divmod_eq(expected_asset_amount, param::SETTLEMENT_PER_CONTRACT, option_token_amount);
+    divmod_eq(expected_asset_amount, param::SETTLEMENT_PER_CONTRACT, grantor_token_amount);
```

Comment:
This last line is redundant since in line 117 we have already asserted `option_token_amount` is equal to `grantor_token_amount`.

Reply:
Fixed